### PR TITLE
Reintroducing nightwatch and fixing tests

### PR DESCRIPTION
**Pull request by EdwinGuzman**
This fixes [SIMPLY-1927](https://jira.nypl.org/browse/SIMPLY-1927). I'm breaking this up into two PRs. This one is just to be able to run the tests and not error out (a few tests fail but at least the script doesn't completely break).

Many assertions are commented out but that's because a lot of the DOM has changed and the selectors are not relevant anymore. I renamed `titleContains.js` to `titleContains-old.js` because it seems this is now a native function that does not need to be added but I will double check. The second PR will involve updating any more functions/patterns and fixing and cleaning the tests.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 ## Changelog
 
+### v0.2.1
```
Comment (aslagle) [inline CHANGELOG.md]: I don't think this needs a version bump since the changes only affect tests and the published package should be the same.

**Comment by aslagle** [comment]
It runs for me 👍

File: CHANGELOG.md (hunk 1 above)
Comment (EdwinGuzman) [reply CHANGELOG.md]: That's true but I would like to keep track of all the changes that go into master. Perhaps I can just add a note that this version will not be published to npm?

Comment (aslagle) [reply CHANGELOG.md]: Maybe we can just combine with another change. You can merge this and I'll add on my next PR?

Comment (EdwinGuzman) [reply CHANGELOG.md]: Sounds good!

**Pull request merged**